src/Entity/Entity.php

```diff
@@ -73,6 +73,7 @@ abstract class Entity
      * The last modification time stamp (in UTC) for this Entity.
      *
      * @var \DateTime $modificationTimeStamp;
+     * @Serializer\Groups({"search", "card"})
```

Comment:
Is there a way to do this just for the class that extends this base class attribute. I really don't want to expose it for everything.